### PR TITLE
Random colorize pipe

### PR DESCRIPTION
**Pull request by valakis**

### CODE REVIEW

File: lib/src/pipes/random_colorize_pipe.dart [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class SkawaRandomColorizePipe implements PipeTransform {
+  String transform(dynamic seed) {
+    if (!_supportedInput(seed)) {
+      throw new InvalidPipeArgumentException(SkawaRandomColorizePipe, seed);
+    }
+    num hash;
+    if (seed is String) {
+      hash = seed.hashCode % 1000000;
+    } else if (seed is num) {
+      hash = seed % 1000000;
```
Comment (thomas892) [inline lib/src/pipes/random_colorize_pipe.dart]: This won't return a valid css color if there are less than 6 digits in the seed.

File: test/random_colorize_pipe_test.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      expect(() => pipe.transform(new Object()), throwsAnInvalidPipeArgumentException);
+    });
+    test('returns same color for the same int seed', () {
+      String a = pipe.transform(intSeed);
+      String b = pipe.transform(intSeed);
+      expect(a, allOf(const isInstanceOf<String>(), b));
+    });
+    test('returns valid css color', () {
+      String a = pipe.transform(intSeed);
+      expect(a, allOf(startsWith('#'), hasLength(lessThanOrEqualTo(7))));
```
Comment (thomas892) [inline test/random_colorize_pipe_test.dart]: It has to be equal to either 7 or 4.

Comment (valakis) [reply test/random_colorize_pipe_test.dart]: 5 or 9 also valid (and correctod corresponding to this)

File: lib/src/pipes/random_colorize_pipe.dart (hunk 1 above)
Comment (valakis) [reply lib/src/pipes/random_colorize_pipe.dart]: due to the  mdn [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/color) it is still a valid color if its 3,4 6 or 8 digits number and changed to code to correspondig to that

**Comment by valakis** [comment]
Also added support for hexadecimal numbers

**Pull request merged**